docs/index.html

```diff
@@ -39,680 +39,548 @@
-        label: "Terraform",
-        active: false,
-        link: "../infrastructure/terraform.html",
         moved: 0
       },
       {
         quadrant: 1,
         ring: 0,
         label: "Kubernetes",
         active: false,
-        link: "../infrastructure/kubernetes.html",
         moved: 0
       },
       {
         quadrant: 1,
         ring: 0,
         label: "AWS CloudFront",
         active: false,
-        link: "../infrastructure/aws_cloud_front.html",
         moved: 0
       },
       {
         quadrant: 1,
         ring: 0,
         label: "Kibana",
         active: false,
-        link: "../infrastructure/kibana.html",
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
-        label: "Bareos",
-        active: false,
-        link: "../infrastructure/bareos.html",
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
-        label: "Kerberos",
-        active: false,
-        link: "../infrastructure/kerberos.html",
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
-        label: "Atlantis",
-        active: false,
-        link: "../infrastructure/atlantis.html",
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
-        label: "Cisco ACI",
-        active: false,
-        link: "../infrastructure/cisco_aci.html",
         moved: 0
       },
       {
         quadrant: 1,
         ring: 1,
-        label: "CentOS",
+        label: "RHEL UBI",
         active: false,
-        link: "../infrastructure/centos.html",
         moved: 0
       },
       {
         quadrant: 1,
-        ring: 1,
-        label: "RHEL UBI",
+        ring: 0,
+        label: "OpsGenie",
         active: false,
-        link: "../infrastructure/rhel_ubi.html",
         moved: 0
       },
       {
         quadrant: 1,
         ring: 0,
-        label: "OpsGenie",
+        label: "CocoaPods",
         active: false,
-        link: "../infrastructure/ops_genie.html",
         moved: 0
       },
       {
         quadrant: 1,
         ring: 0,
-        label: "Nginx",
+        label: "Gradle",
         active: false,
-        link: "../infrastructure/nginx.html",
         moved: 0
       },
       {
         quadrant: 1,
         ring: 0,
-        label: "CocoaPods",
+        label: "New Relic",
         active: false,
-        link: "../infrastructure/cocoa_pods.html",
         moved: 0
       },
       {
         quadrant: 1,
-        ring: 0,
-        label: "Gradle",
+        ring: 1,
+        label: "Kong",
         active: false,
-        link: "../infrastructure/gradle.html",
         moved: 0
       },
       {
         quadrant: 1,
         ring: 1,
-        label: "Kong",
+        label: "Mapbox",
         active: false,
-        link: "../infrastructure/kong.html",
         moved: 0
       },
       {
         quadrant: 1,
-        ring: 0,
-        label: "AWS Lambda",
+        ring: 3,
+        label: "Static Maps",
```

Comment:
this is a product / feature of ours not really a technology, isn't it?

Reply:
Yes, but I think with since Mapbox is now also on the radar and both were suggested to be added I would prefer to keep it for now